catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogPropertiesMetadata.java

```diff
@@ -88,10 +88,11 @@ public class IcebergCatalogPropertiesMetadata extends BaseCatalogPropertiesMetad
   static {
     List<PropertyEntry<?>> propertyEntries =
         ImmutableList.of(
-            enumImmutablePropertyEntry(
+            enumPropertyEntry(
                 CATALOG_BACKEND,
                 "Iceberg catalog type choose properties",
                 true,
+                false,
```

Comment:
seems no need to change Iceberg catalog backend.

Reply:
What if the user mistakenly write `HIVE` to `HIEV`?

Reply:
What if a user mistakenly write `Hive` with `hiev`?

Reply:
I shouldn't be worked because the constraint of IcebergCatalogBackend class

Reply:
OK